File: marvelHeroes/src/main/java/com/futureworkshops/marvelheroes/data/network/rx/scheduler/SchedulersProvider.java
/*
 * Copyright (c) 2018 FutureWorkshops. All rights reserved.
 */

package com.futureworkshops.marvelheroes.data.network.rx.scheduler;

import io.reactivex.Scheduler;

public interface SchedulersProvider {

    Scheduler io();

    Scheduler ui();
}

File: marvelHeroes/src/main/java/com/futureworkshops/marvelheroes/data/network/rx/scheduler/SchedulersProvider.kt
/*
 * Copyright (c) 2018 FutureWorkshops. All rights reserved.
 */

package com.futureworkshops.marvelheroes.data.network.rx.scheduler

import io.reactivex.Scheduler

interface SchedulersProvider {
    
    fun io(): Scheduler
    
    fun ui(): Scheduler
}
